stdlib/tkinter/__init__.pyi

```diff
@@ -3504,32 +3504,32 @@ class LabelFrame(Widget):
         cnf: dict[str, Any] | None = {},
         *,
         background: str = ...,
-        bd: _ScreenUnits = ...,
+        bd: _ScreenUnits = 2,
         bg: str = ...,
         border: _ScreenUnits = ...,
```

Comment:
`border` is an alias for `borderwidth`, so it should have the same default.

The way short names work is a bit surprising. Anything like `borderwi` would be a valid alias for `borderwidth`. I added `border` to the stubs because I have some code that passes it with that name, probably because I copied it from somewhere.